README.org

```diff
@@ -32,6 +32,15 @@ Optionally you can install it via [[https://github.com/quelpa/quelpa][quelpa]] t
   (justl-recipe-width 25))
 #+end_example
 
+For [[https://github.com/doomemacs/doomemacs][doom emacs]], you can install as:
```

Comment:
Is this primarily for doom emacs or for integration with evil mode ?  I'm slightly confused as I have no idea about either of them. :-)

Reply:
That particular change is only for doom emacs. I'll try to eventually (tm) create PR in evil-collection.